src/components/SiteNav/styles.css

```diff
@@ -1,21 +1,46 @@
-#site-nav {
-	padding: var(--size-2);
+#desktop-navigation, #mobile-navigation {
 	background-color: var(--tjs-dark-bg);
 	font-weight: bold;
+	padding: var(--size-2);
+}
+
+#hamburger-button {
+	background-color: transparent;
+	padding: var(--size-3)
+}
+
+#mobile-navigation {
+	display: none;
 }
 
-#site-nav ul {
-	gap: var(--size-2);
-	align-items: center;
-	padding: var(--size-3);
+#mobile-menu:popover-open {
+	width: 100%;
+	height: 100%;
```

Comment:
Please double check how this behaves on mobile and with 200% zoom. That is a usual zoom level for people with impaired vision and a good test parameter.